### PR TITLE
Added wrapper for PrintReadsSpark

### PR DESCRIPTION
**Pull request by fgvieira**

### CODE REVIEW

File: bio/gatk/printreadsspark/wrapper.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+)
+spark_extra = snakemake.params.get("spark_extra", "")
+java_opts = get_java_opts(snakemake)
+
+
+shell(
+    "gatk --java-options '{java_opts}' PrintReadsSpark {extra} "
+    "--reference {snakemake.input.ref} --input {snakemake.input.bam} "
+    "--output {snakemake.output.bam} "
+    "-- --spark-runner {spark_runner} --spark-master {spark_master} {spark_extra} "
```
Comment (jafors) [inline bio/gatk/printreadsspark/wrapper.py:25]: What does the solitary `--` stand for here? Is it needed before the spark parameters?

**Review by jafors — COMMENTED** [review]
Thanks again! I have one question below.

File: bio/gatk/printreadsspark/wrapper.py (hunk 1 above)
Comment (fgvieira) [reply bio/gatk/printreadsspark/wrapper.py:25]: Yes, it is the way to separate between PrintReads and spark parameters, just like in:

https://github.com/snakemake/snakemake-wrappers/blob/07a6d399384dcf58465825a99b1aa5d79f434769/bio/gatk/baserecalibratorspark/wrapper.py#L23-L29

File: bio/gatk/printreadsspark/environment.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - gatk4 ==4.2.0.0
+  - openjdk =8
+  - snakemake-wrapper-utils ==0.2.0
```
Comment (dlaehnemann) [inline bio/gatk/printreadsspark/environment.yaml:8]: Do these need to be pinned down to these versions, or would newer patch versions also be fine to include (to get bug fixes in). I.e. something like:

```suggestion
  - gatk4 =4.2
  - openjdk =8
  - snakemake-wrapper-utils =0.2
```

Compare to the conda version matching specs:
https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#build-version-spec

File: bio/gatk/printreadsspark/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+name: gatk PrintReadsSpark
+description: |
+  Run gatk PrintReadsSpark.
```
Comment (dlaehnemann) [inline bio/gatk/printreadsspark/meta.yaml:3]: Maybe make this describe what PrintReadsSpark actually does?
```suggestion
  Write reads from SAM format file (SAM/BAM/CRAM) that pass specified criteria to a new file. This is the version that can be run on Spark.
```

**Review by dlaehnemann — APPROVED** [review]
Thanks a bunch!

Two minor things. Feel free to submit those suggestions or not.

**Pull request merged**